compiler/plugins/target/LLVMCPU/LLVMTargetOptions.cpp

```diff
@@ -333,9 +259,18 @@ LLVMTarget::loadFromConfigAttr(Location loc, DictionaryAttr config,
                         "accompanied by 'target_triple'";
       return {};
     }
+    ResolveCPUAndCPUFeaturesStatus status;
     std::optional<LLVMTarget> maybeTarget =
-        LLVMTarget::create(*triple, cpu ? *cpu : "generic",
-                           cpuFeatures ? *cpuFeatures : "", linkEmbedded);
+        LLVMTarget::create(*triple, cpu.value_or(""), cpuFeatures.value_or(""),
+                           linkEmbedded, status);
+    if (status != ResolveCPUAndCPUFeaturesStatus::OK) {
+      // TODO(bjacob): after people have had time to adapt, typically by adding
```

Comment:
Link the issue (https://github.com/iree-org/iree/issues/18561) in the TODO

```suggestion
      // TODO(#18561): after people have had time to adapt, typically by adding
```